src/lang/codeManager.ts

```diff
@@ -57,6 +57,10 @@ export default class CodeManager {
     return this._code
   }
 
+  localStoragePersistCode(): string {
```

Comment:
This sounds like a method to persist some code to localStorage, but it just gets from it. Could it be a getter?